config/kubernetes/base/operator-config.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: operators-info
```

Comment:
Thinking about OpenShift, since this is going to be created in `openshift-operators` where other operators exist as well, will the name `operators-info` not be confusing? How about renaming it to `tekton-operator-info` or something that's more descriptive?

Reply:
yes I agree, but in that case we'll have to think about the name for pipelines version configMap as well because
the configMap name is `pipelines-info` in `openshift-pipelines` namespace
wdyt ?

Reply:
... maybe? I think the broader problem is that there can be other component's operators installed in `openshift-operators` namespace as well and `operators-info` seems like a generic name for all the operators. About `pipelines-info`, it at least belongs to OpenShift Pipelines so it's not as confusing maybe?

Reply:
yep that make sense as well, just had a question, would it break the consistency, i.e. all components have name as `<component's-name>-info` ?

cc @nikhil-thomas

Reply:
good point, we can consider naming all the configmaps to `tekton-<component>-info` as well 🤔

Reply:
for all the other components, as the config-maps live in the targetNamespace, we can assume that it will create less confusion. For operator this could be confusing especially on OpenShift as we developers or users cannot change the installation namespace (because OLM expects our operator to be in openshift-operators.. long story 🤓 ). 

I think in the scope of this pr, let us rename the operator configmap to `tekton-operator-info`. 
cc @piyush-garg

Reply:
> good point, we can consider naming all the configmaps to tekton-<component>-info as well 🤔

we can. 
However, there was a community wide TEP to get consensus on the current naming 🤓 . So we might have to take the TEP path if we want to change it. The reason is the `tkn version` command depend upon these names.

[TEP-0041: Tekton Component Versioning](https://github.com/tektoncd/community/blob/main/teps/0041-tekton-component-versioning.md)

Reply:
i think this is another `namespaces are one honking great idea` moment 😇